lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -43,14 +49,20 @@ function multiplyFractions(node) {
   });
 
   const newNumerator = Node.Creator.parenthesis(
-    Node.Creator.operator('*', numeratorArgs));
+    Node.Creator.operator('*', numeratorArgs)
+  );
+
   const newDenominator = denominatorArgs.length === 1
     ? denominatorArgs[0]
     : Node.Creator.parenthesis(Node.Creator.operator('*', denominatorArgs));
 
   const newNode = Node.Creator.operator('/', [newNumerator, newDenominator]);
+
+  // Apply cancellation
+  const simplifiedNode = cancelLikeTerms(newNode)['newNode'];
+
   return Node.Status.nodeChanged(
-    ChangeTypes.MULTIPLY_FRACTIONS, node, newNode);
+    ChangeTypes.MULTIPLY_FRACTIONS, node, simplifiedNode);
```

Comment:
there's an issue here - now even if the main change is cancelling, it'll say that the changeType is multiplying fractions. pedagogically there should always be a Node.Status with `ChangeTypes.MULTIPLY_FRACTIONS` when fractions are multiplied without cancelling, and we probably want a Node.Status with just the cancelling changeType and no multiplying fractions one if *do* cancel - does that seem right?

also I think the changegroups will be messed up here - double check that the right nodes (the ones being cancelled) are still highlighted in the `oldNode` 

this can be fixed (I think?) by just returning the status from `cancelLikeTerms(newNode)` if a change happens there